ui/org.eclipse.tools.layout.spy/build.properties

```diff
@@ -21,3 +21,4 @@ bin.includes = META-INF/,\
                about.html,\
                icons/
 src.includes = about.html
+additional.bundles = org.eclipse.jdt.annotation
```

Comment:
This is not correct to handle the situation, see 
- https://github.com/eclipse-pde/eclipse.pde/issues/1424#issuecomment-2382669427
use `jars.extra.classpath` instead!